modules/apps/object/object-test/src/testFunctional/macros/ObjectRelationships.macro

```diff
@@ -0,0 +1,23 @@
+definition {
+
+	macro assertRelationshipParameterNotPresent {
+		Click(locator1 = "ObjectRelationships#SELECT_PARAMETER_FORM");
+
+		AssertElementNotPresent(
+			key_parameterOption = "${parameter}",
+			locator1 = "ObjectRelationships#SELECT_PARAMETER_OPTION");
+	}
+
+	macro editRelationshipsParameter {
+		Click(locator1 = "ObjectRelationships#SELECT_PARAMETER_FORM");
+
+		AssertElementPresent(
+			key_parameterOption = "${parameter}",
+			locator1 = "ObjectRelationships#SELECT_PARAMETER_OPTION");
+
+		ClickNoError.clickNoMouseOver(
+			key_parameterOption = "${parameter}",
+			locator1 = "ObjectRelationships#SELECT_PARAMETER_OPTION");
+	}
```

Comment:
We don't need `AssertElementPresent` here since `ClickNoError.clickNoMouseOver` will fail if the element isn't present. Also, I want us to start moving our assertions out of macros and put them in the test level. I'll explain more next week.